src/main/resources/swagger/rawls.yaml

```diff
@@ -3021,6 +3021,37 @@ paths:
             - email
             - profile
 
+  '/api/user/billingAccounts':
+    get:
+      responses:
+        '200':
+          description: Successful Request
+          schema:
+            type: array
+            description: list of billing accounts for this user
+            items:
+              $ref: '#/definitions/BillingAccount'
+        '403':
+          description: Forbidden. You probably missed a scope; more details in the ErrorReport's message field
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        '500':
+          description: Rawls Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      description: list billing accounts for a user
+      tags:
+        - billing
+        - users
+      summary: list billing accounts for a user
+      operationId: listUserBillingAccounts
+      security:
+        - authorization:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/cloud-platform
```

Comment:
if the API is using cloud-billing (not cloud-platform), swagger should use the same